dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -865,7 +867,7 @@ private void startNewReadRequestSpan()
             readRequestCount++;
 
             currentReadRequestEvent = new ReadRequestEvent();
-            currentReadRequestEvent.requestTime = System.nanoTime();
+            currentReadRequestEvent.requestTime = System.currentTimeMillis();
```

Comment:
do we know why nano time doesn't work

Reply:
I believe it was getting truncated by something in the OTEL stack